### ISSUE TITLE
Duplicated code to find player resource nodes

### ISSUE THREAD

**Issue by Malcolmnixon** [GodotVR/godot-xr-tools#70]
There are numerous almost-identical routines to find player resource nodes:
 - Find the ARVROrigin from a node under the player
 - Find the ARVRCamera from a node under the player
 - Find the PlayerBody from a node under the player

MovementProvider.gd has some of these functions, but they only work if the caller is a MovementProvider. Instead it would be better if these find routines were available to any node under the player - for example by making them static functions of a helper script.

**Comment by BastiaanOlij** [comment]
It would indeed be nice to have a number of these functions in an autoload script, my main worry here is that this requires a setup action and I want as much as possible to remain automatic.

I wish GDScript had an include function :)

**Comment by Malcolmnixon** [comment]
I added a pull request showing an example of what I'm thinking of. It lets a node located anywhere under the ARVROrigin do:
```
    var origin := ARVRHelpers.get_arvr_origin(self)
    var camera := ARVRHelpers.get_arvr_camera(self)
    var left_controller := ARVRHelpers.get_left_controller(self)
    var right_controller := ARVRHelpers.get_right_controller(self)
```

**Comment by BastiaanOlij** [comment]
Yeah 100% agree with what it does, just thinking of how we can have the helper node there or auto loaded so the user of the plugin doesn't need to be instructed to do so before starting to use the plugin.
